frontend/packages/console-plugin-sdk/src/utils/nodejs-modules.ts

```diff
@@ -0,0 +1,5 @@
+export const reloadModule = (request: string) => {
+  delete require.cache[require.resolve(request)];
+  // eslint-disable-next-line
+  return require(request);
+};
```

Comment:
The `reloadModule` function is called in `getExecutableCodeRefSource` function when transforming `EncodedCodeRef` into `CodeRef` function source. We import the requested (exposed) module in order to validate that it has the requested export.

```ts
const importPath = `${pkg.name}/${exposedModules[moduleName]}`;
let requestedModule: object;

try {
  requestedModule = reloadModule(importPath);
} catch (error) {
  validationResult.addError(`Cannot import '${importPath}' ${errorTrace}`);
  return emptyCodeRefSource;
}
```

When running webpack dev-server, `ConsoleActivePluginsModule` adds all exposed module files (`consolePlugin.exposedModules` in `package.json`) to the watch list. When the content of an exposed module file changes, we want to validate the up-to-date module exports, so that's why `reloadModule` first clears the cache entry for requested file before importing it via `require`.